src/FormBuilder/resources/js/form-builder/src/blocks/fields/amount/inspector/index.tsx

```diff
@@ -102,6 +103,17 @@ const Inspector = ({attributes, setAttributes}) => {
                             : __('The donation amount is fixed to the following amount:', 'give')
                     }
                 />
+                {priceOption === 'multi' && (
+                    <SelectControl
+                        label={__('Default Level', 'give')}
+                        options={levels.map((level, index) => ({
+                            label: formatCurrencyAmount(level),
+                            value: level,
+                        }))}
+                        value={defaultLevel}
+                        onChange={(defaultLevel) => setAttributes({defaultLevel})}
```

Comment:
Since people have to the ability to update the donation levels list - I think we'll additionally need to programmatically update this value on donation level change to make sure the default value exists in the donation levels array.  Otherwise there's a chance this default level could be saved but not have a matching level.